sherpa/utils/__init__.py

```diff
@@ -63,8 +63,19 @@
 
 try:
     import multiprocessing
-    _multi = True
+    try:
+        multiprocessing_start_method = config.get('multiprocessing', 'multiprocessing_start_method')
```

Comment:
Do we need the try/except - can we not just say

```
blah = config.get('multiprocessing', 'long name', fallback='fork')
```

or does the fallback option not catch all error cases (if so it would mean looking at the other config.get calls we have).

Reply:
The `config.get` docstring was sufficiently confusing that I wasn't sure about `fallback`. Assuming that works even with a missing section then definitely :+1:.

Reply:
@DougBurke  fallback option looks good. I'll update the code in a few min.